easybuild/easyconfigs/b/Bazel/Bazel-0.4.4.eb

```diff
@@ -0,0 +1,31 @@
+# @author: Robert Schmidt (OHRI)
+# @author: Guilherme Peretti-Pezzi (CSCS) 
+
+easyblock = "CmdCp"
+
+name = 'Bazel'
+version = '0.4.4'
+
+homepage = 'http://bazel.io/'
+description = """Bazel is a build tool that builds code quickly and reliably. 
+It is used to build the majority of Google's software."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['%(namelower)s-%(version)s-dist.zip']
+source_urls = ['https://github.com/bazelbuild/bazel/releases/download/%(version)s']
+
+dependencies = [
+    ('Java', '1.8.0_121'),
+]
+
+cmds_map = [('.*', "JAVA_VERSION=%(javashortver)s CC=gcc CXX=g++ ./compile.sh")]
+
+files_to_copy = [(['output/bazel'], 'bin')]
+
+sanity_check_paths={
+    'dirs':  ['bin'],
+    'files': ['bin/bazel']
```

Comment:
@rjeschmi nitpicking, what I was going for is:

```python
sanity_check_paths = {
    'files': ['bin/bazel'],
    'dirs': [],
}
```

i.e. , there's no point in checking for a non-empty `bin` dir if you're also checking for `bin/bazel`

Style-wise: keep the `files/dirs` order like we do in other easyconfigs, use spaces around `=`.